en-US/win10/SupportedInterfaces/USBDevices.md

```diff
@@ -52,4 +52,5 @@ Chalkboard Electronics 7" LCD Capacitive Touchscreen Display | RPI2
 Sierra Wireless Beam (AirCard 340U)                          | MBM               | Sierra Wireless Beam (AirCard 340U) 4G LTE USB Mobile Broadband Modem                                                                                                                 |                                                                                                                                                                                                                                                                                                                                                                          |                                                                                                                                                                                                                                 |![Verified][MSVerified]{:.MsVerified}
 Microsoft Xbox 360 Controller                                | RPI2              | An HID-compliant USB gamepad for Microsoft's Xbox 360                                                                                                                                 | Access using [HidDevice]('https://msdn.microsoft.com/en-us/library/windows/apps/windows.devices.humaninterfacedevice.hiddevice.aspx') (not [Gamepad](https://msdn.microsoft.com/en-us/library/windows/apps/windows.gaming.input.gamepad.aspx))                                                                                                                         | [Robot Kit](https://microsoft.hackster.io/en-US/windowsiot/robot-kit-6dd474)                                                                                                                                                    |![Verified][MSVerified]{:.MsVerified}
+MyTeletouch                                | RPI2              | An HID-compliant USB wireless mouse, keyboard and gamepad                                                                                                                                 | Access using your phone (Iphone, Android, Windows)                                                                           | [Product Webpage](https://www.myteletouch.com)                                                                                                                                                    |![Verified][MSVerified]{:.MsVerified}
```

Comment:
Hi Thanks for your prompt response. I'm not MS employee so I have removed '![Verified][MSVerified]{:.MsVerified} ' and -s from the https. I will be happy to send one piece of MyTeletouch if someone at MS is interested to play with it and test.

Regards
Julian